recycleIT-frontend/src/Components/Header/Header.tsx

```diff
@@ -44,123 +56,152 @@ const Header:React.FC = () => {
-                    <Button onClick={handleCloseNavMenu} sx={{ my: 2, color: 'white', display: 'block' }}>
-                      HOME
-                    </Button>
-                  </Link>
-                  <Link to="/services" className={styles.Link}>
-                    <Button onClick={handleCloseNavMenu} sx={{ my: 2, color: 'white', display: 'block' }}>
-                      SERVICES MAP
-                    </Button>
-                  </Link>
-                  <Link to='/support' className={styles.Link}>
-                    <Button onClick={handleCloseNavMenu} sx={{ my: 2, color: 'white', display: 'block' }}>
-                      SUPPORT
-                    </Button>
-                  </Link>
-              </Box>
-              <Box className={styles.Header_avatar_wrapper}>
-                  <Tooltip title="Open settings">
-                    <IconButton onClick={handleOpenUserMenu} sx={{ p: 0 }}>
-                        <Avatar alt="Remy Sharp" src={AccountCircleFilled} className={styles.Header_account_avatar} />
-                    </IconButton>
-                  </Tooltip>
-                  <Menu
-                    sx={{ mt: '45px' }}
-                    id="menu-appbar"
-                    anchorEl={anchorElUser}
-                    anchorOrigin={{
-                        vertical: 'top',
-                        horizontal: 'right',
-                    }}
-                    keepMounted
-                    transformOrigin={{
-                        vertical: 'top',
-                        horizontal: 'right',
-                    }}
-                    open={Boolean(anchorElUser)}
-                    onClose={handleCloseUserMenu}
-                    >
-                    {settings.map((setting) => (
-                        <MenuItem key={setting} onClick={handleCloseUserMenu}>
-                        <Typography textAlign="center">{setting}</Typography>
-                        </MenuItem>
-                    ))}
-                  </Menu>
-                  <Link to="/authorization">
-                    <Avatar src={LogoutOutlined} alt='garbageRecycle'className={styles.Header_logoOut}/>
-                  </Link>
-              </Box>
-            </Box>
-            <IconButton
-              size="large"
-              aria-label="show more"
-              className={styles.Header_burger}
-              aria-haspopup="true"
-              onClick={handleMobileMenuOpen}
-              color="inherit"
-            >
-              <img src={burger} alt="burger" />
-   
+          <Link to='/'>
+            <IconButton>
+                <img src={IconLogo} alt="logo"></img>
             </IconButton>
-            {renderMobileMenu}
+          </Link>
+            {
+              (location.pathname !== '/authorization') ? renderMenu : null
+            }
+            {
+              (location.pathname !== '/authorization') ? renderMobileMenu : null
+            }
```

Comment:
There's already a variable in the code called `isMobileDevice`. From what I see now both menus are rendered always. Why not optimize it like so
```
{
  location.pathname !== '/authorization' &&
  (isMobileDevice ? renderMobileMenu : renderMenu)
}
```

So that you only render one menu at a time?